backend/hitas/models/apartment.py

```diff
@@ -109,6 +114,20 @@ def old_hitas_ruleset(self) -> bool:
     def address(self) -> str:
         return f"{self.street_address} {self.stair} {self.apartment_number}"
 
+    @property
+    def is_new(self) -> bool:
+        today = timezone.now().date()
+        if self.completion_date is None or self.completion_date > today:
+            return True
+
+        sales: list["ApartmentSale"] = list(get_many_or_cached_prefetch(self, "sales"))
+        no_sales = len(sales) < 1
+
+        no_first_purchase_or_in_the_future = self.first_purchase_date is None or self.first_purchase_date > today
+        no_first_sale_or_in_the_future = no_sales or sales[0].purchase_date > today
```

Comment:
If we decide to migrate the sales history from the old system, we can remove the `first_purchase_date` from apartment, as that is then the information of the sale and not the apartment, and only leave the other check. But before that, we still use the apartment field to keep track of this, so that check is also needed. Otherwise, all apartments would be considered new, which would complicate testing right now.